admin/bin/qserv-admin.py

```diff
@@ -256,19 +262,10 @@ def _parseCreateNode(self, tokens):
 
         requiredKeys = self.requiredOpts['createNode']
 
-        # parse command, 'type' is optional, rest is required
-        options = {'nodeName': tokens[0], 'type': 'worker'}
-        for opt in tokens[1:]:
-            if '=' not in opt:
-                raise QservAdminException(QservAdminException.BAD_CMD,
-                   "Invalid argument '%s', should be <key>=<value>" % opt)
-            key, value = opt.split('=', 1)
-
-            if key not in requiredKeys:
-                raise QservAdminException(QservAdminException.BAD_CMD,
-                   "Unrecognized option key '%s', possible keys: %s" % \
-                   (key, ' '.join(requiredKeys)))
+        # parse command, 'type' and 'state' are optional, rest is required
+        options = {'nodeName': tokens[0], 'type': 'worker', 'state': NodeState.ACTIVE}
 
+        for key, value in self._parseKVList(tokens[1:], requiredKeys):
```

Comment:
I don't think that qserv-admin does any logging at all. I guess as a standalone app it should be easy to debug it if someone needs that.